### DISCUSSION TITLE
Which JavaScript project should I contribute to for the first time?

### DISCUSSION THREAD

**Discussion by moasko** [community/community#28674] [Other Feature Feedback, Questions, & Ideas] [answered]
I think it's a bit difficult for a beginner in javascript to find a real project to contribute to in order to have development experience.

Do you know any from your experience?

**Comment by JaimeTR** [comment]
Hello, can you help me by giving stars ⭐ to my repository:

◉ https://github.com/JaimeTR/SvenBot

**Comment by lucalves** [comment, marked as answer]
Hello @moasko, how are you? I hope well.

From my experience, I can suggest some tips to start contributing to open source projects:

1. Search for repositories with issues that have the good first issues label. This link may help you: https://github.com/topics/good-first-issue

2. Follow the trending of the repositories, and find one that identifies with and that can help to contribute. This link may help you: https://github.com/trending

3. Even if you don't feel safe, try to solve a problem and open a pull request! Other developers will definitely help you and you will progressively improve.

4. At first it may not seem easy, but believe me, it is simpler than you think. 

😉👋
